packages/workshop-utils/src/compile-mdx.test.ts

```diff
@@ -0,0 +1,83 @@
+import fs from 'fs'
+import os from 'os'
+import path from 'path'
+import { describe, it, expect } from 'vitest'
+import { compileMdx } from './compile-mdx.server.js'
+
+describe('compileMdx title parsing', () => {
+	it('should extract title with backticks correctly', async () => {
+		// Create a temporary MDX file with backticks in title
+		const testMdxContent = `# Title with \`something\` highlighted
+
+This is some content.
+`
+		
+		const tempDir = os.tmpdir()
+		const testFile = path.join(tempDir, 'test-backtick-title.mdx')
+		
+		try {
+			fs.writeFileSync(testFile, testMdxContent)
+			
+			const result = await compileMdx(testFile)
+			
+			// The title should be extracted correctly, preserving the full text
+			expect(result.title).toBe('Title with `something` highlighted')
+		} finally {
+			// Clean up the temporary file
+			try {
+				fs.unlinkSync(testFile)
+			} catch {
+				// Ignore cleanup errors
+			}
+		}
```

Comment:
I don't like this cleanup mechanism. Let's do this instead: https://www.epicweb.dev/better-test-setup-with-disposable-objects